New - Add initiator_name parameter

blackknight36 opened the pull request:
This parameter allows a host to use a custom ISCSI initiator name.
By default this value is set to undef which results in using the
name defined by the iscsi-initiator-utils package.

Commit:
New - Add initiator_name parameter

This parameter allows a host to use a custom ISCSI initiator name.
By default this value is set to undef which results in using the
name defined by the iscsi-initiator-utils package.

jflorian reviewed (changes requested):
I would like to see the default in Hiera (as a `null`), please.  Otherwise I like it.

blackknight36 commented:
I've added the default value as null in hiera and updated the manifest file.  Currently initiator_name has the minimum string length set as 1, I'm not sure if we want to allow this since initiator names are usually longer and should look something like this.

iqn.1994-05.com.redhat:cdec5f99a5e2
